File: src/main/java/kr/ac/kumoh/k20230626/javawebspring/repository/LottoRepository.java
package kr.ac.kumoh.k20230626.javawebspring.repository;

import kr.ac.kumoh.k20230626.javawebspring.model.LottoNumber;

public interface LottoRepository {
    LottoNumber save(LottoNumber lottoNumber);
    LottoNumber find();
}

File: src/main/kotlin/kr/ac/kumoh/k20230626/lotto/repository/LottoRepository.kt
package kr.ac.kumoh.k20230626.lotto.repository

import kr.ac.kumoh.k20230626.lotto.model.LottoNumber
import org.springframework.stereotype.Repository

interface LottoRepository {
    fun save(lottoNumber: LottoNumber) : LottoNumber
    fun find(): LottoNumber?
}